docs/development/local-development.md

```diff
@@ -10,9 +10,9 @@ For example, if you think anything is unclear, or you think something needs to b
 
 You need the following dependencies for local development:
 
-- Git `>=2.33.0`
+- Git `>=2.45.1`
 - Node.js `^18.12.0 || >=20.0.0`
-- pnpm `^8.6.11` (use corepack)
+- pnpm `^9.0.0` (use corepack)
```

Comment:
This new number for pnpm comes from our `package.json`'s `engine` field:

https://github.com/renovatebot/renovate/blob/e3351ba4c64d3169b783471245a5e5ce06d28feb/package.json#L136-L139